.github/workflows/ana_pkg_build_cicd.yml

```
@@ -79,5 +135,20 @@ jobs:
             - BASEDIST=s390x/clefos:7
             - MC_ARCH=s390x
             - GCC_VER=7.5
-          tags: continuumio/c3i-linux-s390x:cos7
+          tags: continuumio/anaconda-pkg-build:linux-s390x,cos7
+          push: github.ref == 'refs/heads/master'
+
+      - name: build-push pkg-builder-s390x to ECR
+        uses: docker/build-push-action@a66e35b9cbcf4ad0ea91ffcaf7bbad63ad9e0229
+        with:
+          context: ./anaconda-pkg-build/linux
+          builder: ${{ steps.buildx.outputs.name }}
+          file: ./anaconda-pkg-build/linux/Dockerfile
+          platform: linux/s390x
+          registry: ${{ steps.login-ecr.outputs.registry }}
+          build-args:
+            - BASEDIST=s390x/clefos:7
+            - MC_ARCH=s390x
+            - GCC_VER=7.5
+          tags: continuumio/anaconda-pkg-build:linux-s390x,cos7
```

Maybe that has to be different, e.g.

```suggestion
          tags: public.ecr.aws/y0o4y9o3/anaconda-pkg-build:linux-s390x,cos7
```

---

Sorry I'm not super familiar with ECR, but is this line doing something similar? https://github.com/ContinuumIO/docker-images/pull/245/files#diff-f4ddf9bb04705331349f6fdb79613aabb3706749f917bf607aebad63a285f4bbR61